lang/guile/Portfile

```diff
@@ -58,9 +59,6 @@ configure.args      CPPFLAGS="-I${prefix}/include" \
                     --disable-error-on-warning \
                     --disable-silent-rules
 
-patchfiles          sierra.patch \
```

Comment:
If you removed patchfiles (probably because they were included upstream already?), you could probably remove the files themselves as well.